opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -140,10 +141,10 @@ public SELF user_logged_in_into_opba_as_opba_user_with_credentials_using_fintech
                 .getQueryParams()
                 .getFirst(REDIRECT_CODE_QUERY);
 
-        ExtractableResponse<Response> response = RestAssured
+        ExtractableResponse<Response> response =  RestAssured
                 .given()
+                    .header(X_REQUEST_ID, DEFAULT_X_REQUEST_ID)
```

Comment:
Consent API calls should use generated request IDs (like it was before)

Reply:
random UUID is generated